tests/suites/test_suite_mpi.data

```diff
@@ -292,6 +594,63 @@ mbedtls_mpi_cmp_mpi:"-1230000000000000000":"":-1
 Test mbedtls_mpi_cmp_mpi: large negative < 0 (1 limb)
 mbedtls_mpi_cmp_mpi:"-1230000000000000000":"0":-1
 
+Base test mbedtls_mpi_core_lt_ct #1
+mbedtls_mpi_core_lt_ct:"02B5":"02B5":0
+
+Base test mbedtls_mpi_core_lt_ct #2
+mbedtls_mpi_core_lt_ct:"02B5":"02B4":0
+
+Base test mbedtls_mpi_core_lt_ct #3
+mbedtls_mpi_core_lt_ct:"02B5":"02B6":1
+
+Base test mbedtls_mpi_core_lt_ct (length=0)
+mbedtls_mpi_core_lt_ct:"":"":0
+
+Base test mbedtls_mpi_core_lt_ct (corner case - 64 bit) #1
+mbedtls_mpi_core_lt_ct:"7FFFFFFFFFFFFFFF":"FF":0
```

Comment:
Please don't use an `mbedtls_xxx` prefix in test function names in new code. The test framework actually prepends `"mbedtls_test_"` to test function names so there's no need for a disambiguation prefix. Having a test function with the same name as the tested function is annoying because source code navigation sees that as ambiguous.

Reply:
> Having a test function with the same name as the tested function is annoying

Sorry, I disagree. I've found it much easier (notably coming into Mbed TLS with no prior experience of it) navigating the test framework when the name of the test is specifically the name of the function being tested.  As you point out, the framework adds a prefix so the C function names don't clash.

Reply:
How does it help to have _exactly_ the same name, as opposed to e.g. the same name without the `mbedtls_` prefix? And often there are differences anyway because there isn't a one-to-one correspondence between library functions and test functions.

Reply:
Beyond helping my brain match an exact string, being able to use `*` in the `.data` file then switch to the `.function` file and type `n` is incredibly quick.

> there isn't a one-to-one correspondence between library functions and test functions

Not directly one-to-one, but every library function should have a corresponding unit test function - and again, having an obvious library function name as the name of the test makes this easier to see - the test names not looking like a library function would help in this case, too

Reply:
It looks like not having `mbedtls_` prefix is much more common:
```
perl -n -e 'print if /^void/' $(find . -name \*.function) | wc -l
     766
perl -n -e 'print if /^void mbedtls_/' $(find . -name \*.function) | wc -l
     168
```

I suggest we go without `mbedtls_` prefix for consistency.

Reply:
The numbers are actually a bit closer

```
$ perl -n -e 'print if /^\w+:/' *.data | fgrep -v depends_on | cut -d: -f1 | sort -u | fgrep mbedtls_ | wc -l
     155
$ perl -n -e 'print if /^\w+:/' *.data | fgrep -v depends_on | cut -d: -f1 | sort -u | fgrep -v mbedtls_ | wc -l
     570
```

(and some of the "no mbedtls_" aren't actual test cases)

but: surely it's just logical to name a unit test case with the same name as the function it's testing?

Clearly the test system adds a prefix precisely so this can be done: so it's at the very least supported.

I suspect once a `.data` file either has or doesn't have test names with the prefix, additional tests in that file will use the same style for consistency.

Trying to figure out which is more recent is... TBH more work than I have time for.